CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [5.1.0]
+### Changed
+- deps: bn.js@4.11.8->4.12.0 ([#309](https://github.com/MetaMask/eth-sig-util/pull/309))
+- Type specification; minor input validation fixes; TypeScript 4.8 ([#307](https://github.com/MetaMask/eth-sig-util/pull/307))
```

Comment:
Nit: It would be great to explain in more detail what these input validation fixes are

We can split this into separate entries as well if it's easier to read that way. The TypeScript update might be more easily read as a separate entry for example